packages/block-library/src/navigation/edit/index.js

```diff
@@ -794,7 +798,18 @@ function Navigation( {
 						{ hasResolvedCanUserDeleteNavigationMenu &&
 							canUserDeleteNavigationMenu && (
 								<NavigationMenuDeleteControl
-									onDelete={ resetToEmptyBlock }
+									onDelete={ ( deletedMenuTitle = '' ) => {
+										resetToEmptyBlock();
+										showNavigationMenuDeleteNotice(
```

Comment:
Before this is merged we should try and handle both success and failure cases. The `onDelete` could handle both by way of a flag in the callback. We should show a different message based on the result.

Reply:
Ok I looked at this and it will require a larger refactor. Let's run with the current happy path and revisit the feedback later.